### PR TITLE
Update setting-up-your-phone-system.md

### PR DESCRIPTION
**Pull request by dariomws306**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/3754

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/setting-up-your-phone-system.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ For more information about call queues, see [Create a Cloud call queue](create-a
 
 ## Step 8: If you want to set up a Cloud auto attendant
 
-Auto attendants let people that call in to your organization and navigate a menu system to get them to the right department, call queue, person, or the operator. You can create an auto attendant for your organization by using the Skype for Business admin center.
+Auto attendants let people that call in to your organization and navigate a menu system to get them to the right department, call queue, person, or the operator. You can create an auto attendant for your organization by using the Teams admin center.
```
Comment (JohanFreelancer9) [inline Teams/setting-up-your-phone-system.md]:
```suggestion
Auto attendants let people that call in to your organization navigate a menu system to get them to the right department, call queue, person, or the operator. You can create an auto attendant for your organization by using the Teams admin center.
```

Comment (dariomws306) [reply Teams/setting-up-your-phone-system.md]: I think it's ok. If we add the word "and" we should remove "that":

Auto attendants let people call in to your organization and navigate..........

Comment (JohanFreelancer9) [reply Teams/setting-up-your-phone-system.md]: @dariomws306 Agree.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**